tonic-build/src/lib.rs

```diff
@@ -144,6 +144,8 @@ pub trait Method {
     fn server_streaming(&self) -> bool;
     /// Get comments about this item.
     fn comment(&self) -> &[Self::Comment];
+    /// Method is deprecated.
+    fn deprecated(&self) -> bool;
```

Comment:
Could we give this a default implementation that yields `false`? That would make this a non-breaking change, which might be nice, and it seems conceptually reasonable.

Reply:
For now I agree to make this setting false by default, and I think it would be good to set it to true in the future breaking release.

Reply:
That's a good idea, I've added it

Reply:
> For now I agree to make this setting false by default, and I think it would be good to set it to true in the future breaking release.

Why? That doesn't make much sense to me.

Reply:
What I intended to mean was to not mark the code generated from the proto's deprecated as deprecated by default this time, and to enable it in the next major release. And I just noticed that I was commenting on something different than the perspective of your review.

Reply:
Ah, so you mean removing the default implementation when we release 0.13? That makes sense to me.